src/Kunstmaan/MediaBundle/Controller/MediaController.php

```diff
@@ -314,20 +323,30 @@ public function dropAction(Request $request, $folderId)
                 $drop = $request->get('text');
             }
         }
-        $media = $this->get('kunstmaan_media.media_manager')->createNew($drop);
+        $mediaManager = $this->get('kunstmaan_media.media_manager');
+        $media = $mediaManager->createNew($drop);
         if ($media) {
             $media->setFolder($folder);
             $em->getRepository('KunstmaanMediaBundle:Media')->save($media);
 
             return new JsonResponse(['status' => $this->get('translator')->trans('kuma_admin.media.flash.drop_success')]);
         }
 
+        if ($mediaManager->isExtensionAllowed($drop)) {
```

Comment:
Should this be `false === $mediaManager->isExtensionAllowed($drop)`?

Reply:
After reading the code again I see what is happening here, maybe we should turn it around to make it more clear?

Reply:
No I do this second check to keep the old message that would have been returned if a file upload fails because of unknown reasons. To keep the responses like it was and just add my own message if it was specifically not allowed